src/EA.Weee.RequestHandlers/Mappings/AatfMap.cs

```diff
@@ -49,10 +52,13 @@ public AatfData Map(Aatf source)
 
             var organisation = organisationMap.Map(source.Organisation);
 
+            Core.AatfReturn.FacilityType facilityType = facilityMap.Map(source.FacilityType);
```

Comment:
Add a unit tests to test that the mapper is called and that the AatfData returned below contains this mapped object